app/src/main/java/ch/epfl/sweng/erpa/activities/DependencyConfigurationAgnosticActivity.java

```diff
@@ -20,7 +20,9 @@
 import toothpick.Toothpick;
 
 public abstract class DependencyConfigurationAgnosticActivity extends Activity {
-    @Inject @Named("Dependency Configurators") Set<DependencyConfigurator> coordinators;
+    @Inject
```

Comment:
Having a line for each annotation makes the code harded to overview and unnesessary long. In general we want to keep groups of annotations (such as normal fields, autoinjected fields, automatic constructors, lombok on the future, ...)

Please modify your editor settings not to manually wrap (except for class annotations).
Please modify the code to have all field member, methods and parameter annotations in the same line